src/argilla/client/feedback/dataset.py

```diff
@@ -581,7 +586,7 @@ def delete_dataset(dataset_id: UUID) -> None:
                     datasets_api_v1.add_records(
                         client=httpx_client,
                         id=argilla_id,
-                        records=[json.loads(record.json()) for record in batch],
+                        records=[json.loads(record.dict()) for record in tqdm(batch)],
```

Comment:
This won't work! `json.loads` expects either a `str` or `bytes`. Can you revert this change?

Reply:
Indeed that's already modified in the #3234 branch as mentioned below! But yes, that would run into conflicts once the other PR is merged into `develop`